admittance_controller:
  ros__parameters:
    ft_sensor_tf: ur5e_ft_sensor
    base_tf: ur5e_base_link
    joint_command_topic: /forward_velocity_controller/command
    ft_wrench_topic: /wrench

    # matrix should be written in row major
    M_matrix: []
    B_matrix: []
    K_matrix: []
